frontend/doc/doxygen-pitfalls.txt

```diff
@@ -12,6 +12,7 @@ Doxygen will fail to resolve:
 
   - C++ standard library types (that are not builtin, e.g. `int64_t`)
   - Any `compiler/dyno` types that are generated by macros (for now)
```

Comment:
Can we remove this line (or should we update it to say we think it should be resolved now)?

Reply:
This line is still accurate and the problem has not been resolved, AFAIK. We use workarounds.

Reply:
Ah, okay.  Thanks!